src/test/java/api/requests/RequestsAPICreationTests.java

```diff
@@ -1328,7 +1328,7 @@ public static IndividualResource setupPagedItem(IndividualResource requestPickup
   }
 
   public static IndividualResource setupItemAwaitingPickup(IndividualResource requestPickupServicePoint, ResourceClient requestsClient, ResourceClient itemsClient,
-                                                           ItemsFixture itemsFixture, UsersFixture usersFixture, LoansFixture loansFixture) {
+                                                           ItemsFixture itemsFixture, UsersFixture usersFixture, CheckInFixture checkInFixture) {
```

Comment:
Whilst this line is being changed, please could the indentation be changed to a standard 2 space line continuation

Reply:
Could the same be done for other methods in this file please.